apps/help-center/package.json

```diff
@@ -41,6 +41,7 @@
 		"@wordpress/data": "^10.8.0",
 		"@wordpress/plugins": "^7.8.0",
 		"@wordpress/private-apis": "^1.8.0",
+		"@wordpress/router": "^1.11.0",
```

Comment:
This being a monorepo I'm not sure if this dep was added correctly. I just did a `yarn add ...` in `apps/help-center/` it seemed to create a `node_modules/` there which is not what I think we want.

Reply:
The `wpcom-block-editor` uses `"@wordpress/router": "^1.8.0",` so it would be better to keep it the same or update both.